src/docfx/build/page/PageBuilder.cs

```diff
@@ -125,16 +128,21 @@ public void Build(ErrorBuilder errors, FilePath file)
             return (new JObject(), new JObject());
         }
 
-        var systemMetadataJObject = JsonUtility.ToJObject(systemMetadata);
-
         if (JsonSchemaProvider.IsConceptual(mime))
         {
+            if (_config.OutputType == OutputType.Json)
```

Comment:
In symmetry with the SDP pipeline, this property would go to `ConceptualModel` over `SystemMetadata`, because `_xrefmap` isn't a metadata in the context of SDP, it is a property in the output json.